examples-generated/rdns/rdns.yaml

```diff
@@ -23,9 +23,7 @@ metadata:
   name: foo
 spec:
   forProvider:
```

Comment:
Some go from matchLabels to ref, some go the other way around

Reply:
the only one I backed out was image because it is a simple string and not an ID number and it was also causing some circular dependencies where images can be based on things that can then use images. if you think it is worth it I can add back in the ones that do not cause the circular deps